clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -172,6 +172,10 @@
     public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
     private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+    /** <code>exclude.internal.topics</code> */
+    public static final String EXCLUDE_INTERNAL_TOPICS_CONFIG = "exclude.internal.topics";
+    private static final String EXCLUDE_INTERNAL_TOPICS_DOC = "Whether messages from internal topics (such as offsets) should be exposed to the consumer.";
```

Comment:
Can you clarify this description a little bit? It seems like it's still possible for the user to subscribe to internal topics explicitly. Also, don't we still return internal topics from the listTopics() API?